src/FormField/ScopeBuilder.php

```diff
@@ -549,7 +568,7 @@ public static function conditionToQuery(Condition $condition): array
                 $value = implode(',', $value);
                 $operator = $map[$operator] ?? 'IN';
```

Comment:
missed this? 3 maybe there are even more placed...

Reply:
Not really as this is the `atk4\data\Scope` version of operators.
Constants in the class represent the VueQueryBuilder operators